lib/scientist/experiment.rb

```diff
@@ -269,4 +272,13 @@ def try(name = nil, &block)
   def use(&block)
     try "control", &block
   end
+
+  # Whether or not to raise a mismatch error when a mismatch occurs.
+  def raise_on_mismatches?
+    if @raise_on_mismatches.nil?
```

Comment:
Now with the accessor in place this could also be `if raise_on_mismatches.nil?`.